Makefile

```diff
@@ -277,6 +277,11 @@ storage-protos-compatible: $(PROTOLOCK_BIN)
 	@echo "+ $@"
 	$(SILENT)$(PROTOLOCK_BIN) status -lockdir=$(BASE_DIR)/proto/storage -protoroot=$(BASE_DIR)/proto/storage --uptodate true
 
+.PHONY: update-storage-protolock
+update-storage-protolock: $(PROTOLOCK_BIN)
+	@echo "+ $@"
+	$(SILENT)$(PROTOLOCK_BIN) commit -lockdir=$(BASE_DIR)/proto/storage -protoroot=$(BASE_DIR)/proto/storage
```

Comment:
Sorry for the late comment, but I just ran into a failing protolock linter.
Two ideas: 

1. I would strongly welcome a custom liner error message that tells me to run this makefile target - that would make troubleshooting quicker.

2. Why not call this target as part of `make proto-generated-srcs`? That would eliminate a manual step for devs and also minimize the chance that updated protos without a lock get checked in.